app/helpers/is-desired-field-type.js

```diff
@@ -0,0 +1,10 @@
+import { helper } from '@ember/component/helper';
+
+export function isDesiredFieldType(params) {
```

Comment:
Again .... `is-desired-field-type` is unclear as to what this helper does. Wouldn't something like `is-input-field` tell one exactly what this helper does ?